src/node_file.cc

```diff
@@ -3385,7 +3429,8 @@ static void CreatePerIsolateProperties(IsolateData* isolate_data,
   SetMethod(isolate, target, "accessSync", AccessSync);
   SetMethod(isolate, target, "close", Close);
   SetMethod(isolate, target, "closeSync", CloseSync);
-  SetMethod(isolate, target, "existsSync", ExistsSync);
+  SetFastMethodNoSideEffect(
```

Comment:
Don't use `NoSideEffect` here? (see existing and https://github.com/nodejs/node/pull/49857)